test/cypress/integration/confirm/confirm.cy.test.js

```diff
@@ -34,7 +34,7 @@ describe('Confirm page', () => {
       cy.get('[data-cy=summary-list]').get('dt').eq(0).should('contain', 'Total to pay')
       cy.get('[data-cy=summary-list]').get('dd').eq(0).should('contain', '£10.00')
 
-      cy.get('[data-cy=form]').get('#amount').eq(0).should('value', '£10.00')
+      cy.get('[data-cy=form]').get('#amount').eq(0).should('value', '1000')
```

Comment:
For this test, where does this amount come from? Do we expect a different test to have been run first?

Reply:
Good spot - the name of the test was incorrect.  It did not say that the price comes from `product.price`.
This has been fixed.